nym-node-status-api/nym-node-status-api/src/monitor/geodata.rs

```diff
@@ -91,6 +106,19 @@ pub(crate) struct LocationResponse {
     pub(crate) two_letter_iso_country_code: String,
     #[serde(deserialize_with = "deserialize_loc")]
     pub(crate) loc: Coordinates,
+    // TODO dz consider making them optional?
```

Comment:
nit: would it make sense to just wrap all the geodata in separate struct and put it in an `Option` as you suggested yourself?

Reply:
especially since it's the case you will either have everything or nothing at all, i.e. it won't be the case you will have ip but not timezone